translate/src/modules/search/components/SearchPanel.test.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import { createReduxStore, mountComponentWithStore } from '~/test/store';
+
+import { SearchPanel, SearchPanelDialog } from './SearchPanel';
+import { SEARCH_OPTIONS } from '../constants';
+
+describe('<SearchPanelDialog>', () => {
+  it('correctly sets search option as selected', () => {
+    const selectedSearchOptions = {
+      search_identifiers: false,
+      search_exclude_source_strings: false,
+      search_rejected_translations: true,
+      search_match_case: true,
+      search_match_whole_word: false,
+    };
+
+    const store = createReduxStore();
+    const wrapper = mountComponentWithStore(SearchPanelDialog, store, {
+      options: selectedSearchOptions,
+      selectedSearchOptions,
+    });
+
+    for (let option of SEARCH_OPTIONS) {
+      expect(wrapper.find(`.menu .${option.slug}`).hasClass('enabled')).toBe(
+        selectedSearchOptions[option.slug],
+      );
+    }
```

Comment:
You could use the same pattern as below:

```suggestion
    for (let { slug } of SEARCH_OPTIONS) {
      expect(wrapper.find(`.menu .${slug}`).hasClass('enabled')).toBe(
        selectedSearchOptions[slug],
      );
    }
```